pkg/controller/memberstatus/memberstatus_controller.go

```diff
@@ -0,0 +1,305 @@
+package memberstatus
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/member-operator/pkg/configuration"
+	"github.com/codeready-toolchain/member-operator/version"
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+
+	"github.com/go-logr/logr"
+	errs "github.com/pkg/errors"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+	kubefed_v1beta1 "sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
+	"sigs.k8s.io/kubefed/pkg/controller/util"
+)
+
+var log = logf.Log.WithName("controller_memberstatus")
+
+// general memberstatus constants
+const (
+	OperatorNameVar = "OPERATOR_NAME"
+
+	defaultMemberStatusName = "toolchain-member-status"
```

Comment:
I believe that both the `OperatorNameVar` and the `defaultMemberStatusName` could be moved to `configuration.go` file and load it as part of the config

Reply:
In addition, you could use the function `k8sutil.GetOperatorName()`

Reply:
Done and Done in 6f77a0e80d789b3953b2e1269a1d34f1f1effd7d